CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# How to contribute to ESPnet
+
+If you are interested in contributing to ESPnet, your contributions will fall into two categories:
+
+1. You want to propose a new Feature and implement it
+   post about your intended feature, and we shall discuss the design and implementation.
+   Once we agree that the plan looks good, go ahead and implement it.
+        
+2. You want to implement a feature or bug-fix for an outstanding issue
+   Look at the outstanding issues here: https://github.com/espnet/espnet/issues
+   Pick an issue and comment on the task that you want to work on this feature
+   If you need more context on a particular issue, please ask and we shall provide.
+
+Once you finish implementing a feature or bugfix, please send a Pull Request to https://github.com/espnet/espnet
+
+If you are not familiar with creating a Pull Request, here are some guides:
+
+- http://stackoverflow.com/questions/14680711/how-to-do-a-github-pull-request
+- https://help.github.com/articles/creating-a-pull-request/
+
+
+## Developing locally with ESPnet
+
+TBD
+
+## Unit testing
+
+ESPnet's testing is located under `test/`. Run the entire test suite with
+
+``` console
+$ pip install -r tools/test_requirements.txt
```

Comment:
I think this pip install is not required always, and may have some comments about it.